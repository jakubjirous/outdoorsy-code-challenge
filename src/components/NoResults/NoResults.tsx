import React from "react";
import styled from "styled-components";

const S = {
  Wrapper: styled.div`
    display: flex;
    background-color: var(--wisp);
    border-radius: var(--br);
  `,
  Paragraph: styled.p`
    padding: var(--spacing-5);
  `,
};

function NoResults() {
  return (
    <S.Wrapper>
      <S.Paragraph>
        Hmm... We don't have anything available that matches your criteria. Try adjusting your search by changing your
        keywords.
      </S.Paragraph>
    </S.Wrapper>
  );
}

export default NoResults;
