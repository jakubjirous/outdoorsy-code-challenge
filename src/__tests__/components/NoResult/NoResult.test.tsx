import { render } from "@testing-library/react";
import NoResults from "../../../components/NoResults/NoResults";

const renderContainer = () => {
  return render(<NoResults />);
};

describe("<NoResults/>", () => {
  test("should render correctly", () => {
    const { baseElement } = renderContainer();

    expect(baseElement).toMatchSnapshot();
  });
});
