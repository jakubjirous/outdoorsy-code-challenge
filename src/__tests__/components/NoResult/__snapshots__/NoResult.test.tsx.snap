// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoResults/> should render correctly 1`] = `
<body>
  <div>
    <div
      class="sc-bczRLJ ifnEpg"
    >
      <p
        class="sc-gsnTZi gjeoGl"
      >
        Hmm... We don't have anything available that matches your criteria. Try adjusting your search by changing your keywords.
      </p>
    </div>
  </div>
</body>
`;
